modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -679,9 +679,11 @@ else if (value instanceof Object[]) {
 			}
 
 			if (indexType.equals("keyword")) {
+				document.addKeyword(_getSortableFieldName(name), valuesString);
```

Comment:
@carolmariaabb Sorry, I didn't test after sort it.
I don't know why, but doing it the fix doesn't works anymore. Can I return to the original ordination and make clear the order is important here, in the commit message? Or do you have another suggestion?

Reply:
@p-albuquerque please add an empty line to make it clear that order matters:

```
document.addTextSortable(name, valuesString);
				
document.addText(_getSortableFieldName(name), valuesString);
```

Reply:
Copy that

Reply:
There is another adjust I performed related to the unit test. I will push it too